core-rust/state-manager/src/protocol/protocol_configs/mainnet_protocol_config.rs

```diff
@@ -6,7 +6,7 @@ use ProtocolUpdateEnactmentCondition::*;
 pub fn mainnet_protocol_config() -> ProtocolConfig {
     // See config_printer.rs
     ProtocolConfig::new_with_triggers(hashmap! {
-        ANEMONE_PROTOCOL_VERSION => EnactAtStartOfEpochIfValidatorsReady {
+        ProtocolVersionName::anemone() => EnactAtStartOfEpochIfValidatorsReady {
```

Comment:
should we consider updating anomone and bottlenose to `EnactAtStartOfEpochUnconditionally`? :thinking:

Reply:
Could do, but what benefit do you see? Slight win in performance?